_posts/en/newsletters/2022-07-27-newsletter.md

```diff
@@ -0,0 +1,152 @@
+  Stickies-v notes that while provably unspendable UTXOs including `OP_RETURN`
+  or scripts larger than the max script size are removed from the UTXO
+  set, removing [uneconomical outputs][topic uneconomical outputs] could cause
+  issues, including a hard fork as Pieter Wuille points out, if those outputs are spent.
+
+- [Is there code in libsecp256k1 that should be moved to the Bitcoin Core codebase?]({{bse}}114467)
+  Similar to other efforts to modularize areas of the Bitcoin Core codebase like
+  [libbitcoinkernel][libbitcoinkernel project] or [process separation][devwiki
+  process separation], Pieter Wuille notes a clear area of responsibility of the
+  [libsecp256k1][] project: everything that involves operations on private or public keys.
+
+- [Mining stale low-difficulty blocks as a DoS attack]({{bse}}114241)
+  Andrew Chow explains that [assumevalid][assumevalid notes] and more recently
+  [`nMinimumChainWork`][Bitcoin Core #9053] help filter out low-difficulty chain attacks.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding
+
+- BTCPay Server 1.6.2
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #25351][] rpc, wallet: Scan mempool after import - Second attempt FIXME:Xekyo
+
+- [Core Lightning #5370][] reimplements the `commando` plugin and makes
+  it a built-in part of CLN.  Commando allows a node to receive commands
+  from authorized peers using LN messages.  Peers are authorized using
+  *runes*, which is a custom CLN protocol based on a simplified version
+  of [macaroons][].  Although Commando is now built into CLN, it's only
+  operable if a user creates rune authentication tokens.  For additional
+  information, see CLN's manual pages for [commando][] and [commando-rune][].
+
+- [BOLTs #1001][] recommends that nodes who advertise a change to their
+  payment forwarding policies continue accepting payments received
+  using the old policies for about 10 minutes.  This prevents payments
+  from failing just because the sender hasn't heard about a recent
+  policy update.  See [Newsletter #169][news169 cln4806] for the example
+  of an implementation adopting a rule like this.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="25351,5370,1001,24058,9053" %}
+[commando]: https://github.com/rustyrussell/lightning/blob/2e13b72f55080be07ea68de77976eb990a043f5d/doc/lightning-commando.7.md
```

Comment:
```suggestion
[commando]: https://github.com/ElementsProject/lightning/blob/master/doc/lightning-commando.7.md
```

Reply:
I prefer to link to a specific commit so that the link doesn't break if the file is later moved or deleted.